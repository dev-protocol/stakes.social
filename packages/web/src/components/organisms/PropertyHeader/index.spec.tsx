import React from 'react'
import { render } from '@testing-library/react'
import { PropertyHeader } from '.'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/devprtcl/hooks')
jest.mock('src/fixtures/wallet/hooks')

describe(`${PropertyHeader.name}`, () => {
  test('Snapshot', () => {
    const component = render(<PropertyHeader propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
