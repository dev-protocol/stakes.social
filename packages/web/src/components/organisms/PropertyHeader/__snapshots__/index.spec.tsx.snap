// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyHeader Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-hKwDye hPWqws"
    >
      <div
        class="sc-jrQzAO jvAqrf"
      >
        <h1
          class="sc-eCImPb kNcJWM"
        >
          propertyAddress-name Pool
        </h1>
        <div
          class="sc-crHmcD hVJWDK"
        >
          <p>
            <a
              href="https://stakes.social/propertyAddress"
            >
              <img
                alt="Stake to support us"
                src="https://badge.devprotocol.xyz/propertyAddress/descriptive"
              />
            </a>
          </p>
          <div
            class="sc-kDTinF jxzTvV"
          >
            <div
              class="sc-iqseJM dNMBEl"
            >
              <span
                aria-label="copy"
                class="anticon anticon-copy"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="copy"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
                  />
                </svg>
              </span>
            </div>
            <span
              style="font-size: 0.8em; margin-left: 2px;"
            >
              Copy
            </span>
          </div>
        </div>
      </div>
      <p
        class="sc-dkPtRN OyKaL btn-tweet"
      >
        <a
          class="twitter-share-button"
          data-lang="en"
          data-show-count="false"
          data-size="large"
          data-text="Check out propertyAddress-name on Stakes.social #DevProtocol"
          href="https://twitter.com/share"
        >
          Tweet
        </a>
      </p>
      <div
        class="sc-jRQBWg kEnsfn"
      >
        <div
          class="sc-furwcr eIYcYr"
        >
          <div
            class="sc-iCfMLu fEEKua"
          >
            <span
              class="sc-gsDKAQ eTZcOI"
            >
              N/A
              %
            </span>
            <div
              class="sc-gKclnd lbnRHf"
            />
            <span>
              APY Stakers
            </span>
          </div>
          <div
            class="sc-gKclnd lbnRHf"
          >
            |
          </div>
          <div
            class="sc-iCfMLu fEEKua"
          >
            <span
              class="sc-gsDKAQ eTZcOI"
            >
               
              N/A
              %
            </span>
            <div
              class="sc-gKclnd lbnRHf"
            />
            <span>
              APY Creators
            </span>
          </div>
          <div
            class="sc-gKclnd lbnRHf"
          >
            |
          </div>
          <div
            class="sc-iCfMLu fEEKua"
          >
            <span
              class="sc-gsDKAQ eTZcOI"
            >
               
              40,124
            </span>
            <div
              class="sc-gKclnd lbnRHf"
            />
            <span>
              Karma
            </span>
          </div>
        </div>
        <div
          class="sc-pVTFL cFxZFx"
        >
          <a
            class="sc-bdvvtL iCPeoC"
            href="https://app.uniswap.org/#/swap?outputCurrency=0x5caf454ba92e6f2c929df14667ee360ed9fd5b26&use=V2"
            rel="noreferrer"
            target="_blank"
          >
            <span
              style="color: white;"
            >
              Buy DEV
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <script
    src="https://platform.twitter.com/widgets.js"
  />
</body>
`;
